src/blocks/Footer.tsx

```diff
@@ -1,39 +1,47 @@
-import React from 'react'
+import React from 'react';
+import github from '../assets/social-media/github.svg';
+import instagram from '../assets/social-media/instagram.svg';
+import linkedin from '../assets/social-media/linkedin.svg';
```

Comment:
Fajnie byłoby nazywać te ikony w jakiś ustandaryzowany sposób np githubIcon, z dodatkiem "icon", wiemy przynajmniej wewnątrz kodu czy jest to ikona, url, czy może cokolwiek innego.